fluss-server/src/main/java/org/apache/fluss/server/metrics/group/BucketMetricGroup.java

```diff
@@ -37,7 +37,7 @@ public BucketMetricGroup(
             MetricRegistry registry,
             @Nullable String partitionName,
             int bucket,
-            TableMetricGroup parent) {
+            AbstractMetricGroup parent) {
```

Comment:
This is problematic. Because `BucketMetricGroup#getTableMetricGroup` returns the parent as `TableMetricGroup`.